x-pack/plugins/index_management/__jest__/components/index_table.test.js

```diff
@@ -254,15 +264,17 @@ describe('index table', () => {
     expect(button.text()).toEqual('Manage 2 indices');
   });
 
-  test('should show system indices only when the switch is turned on', async () => {
+  test('should show hidden indices only when the switch is turned on', async () => {
     const rendered = mountWithIntl(component);
     await runAllPromises();
     rendered.update();
 
-    snapshot(rendered.find('.euiPagination li').map((item) => item.text()));
-    const switchControl = rendered.find('.euiSwitch__button');
+    snapshot(namesText(rendered));
+
+    const switchControl = findTestSubject(rendered, 'indexTableIncludeHiddenIndicesToggle');
     switchControl.simulate('click');
-    snapshot(rendered.find('.euiPagination li').map((item) => item.text()));
+
+    snapshot(namesText(rendered));
```

Comment:
Let me see what can be done but let's not spend too much time on this (now) as time is scarce. I already improved on a previous bad test to snapshot the list of indices instead of the pagination.
We do have a tech debt to write proper CIT test and get rid of this file.

Reply:
But I agree with you that it is a terrible test, I just try to do with what's in place to avoid breaking other tests and fixing the whole file.